selftests/checkall

```diff
@@ -161,7 +161,7 @@ results_dir_content() {
 
 [ "$SKIP_RESULTSDIR_CHECK" ] || RESULTS_DIR_CONTENT="$(ls $RESULTS_DIR 2> /dev/null)"
 if [ "$TRAVIS" == "true" ]; then
-    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git --enable W0102,W0611 &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
+    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git --enable W0102,W0611,W0612,W0622 &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
```

Comment:
shouldn't this be the last commit (when these are tackled?)

Reply:
Oh no fixes were needed?

Reply:
OK, I see, these were added in previous PR and were only included in `make check`...

Reply:
OK, too tired eyes. This is the only place where we enable warnings and as it's passing there were probably no issues...